service_thing_aggregate/src/main/resources/evolutions/default/1.sql

```diff
@@ -0,0 +1,34 @@
+# --- !Ups
+
+CREATE SCHEMA IF NOT EXISTS MUSIT_MAPPING;
+
+CREATE TABLE MUSIT_MAPPING.VIEW_MUSITTHING (
+ id BIGINT(20) PRIMARY KEY AUTO_INCREMENT,
+ displayId varchar NOT NULL,
+ displayName varchar
+);
+
+CREATE SCHEMA IF NOT EXISTS MUSARK_STORAGE;
+
+CREATE TABLE MUSARK_STORAGE.LOCAL_OBJECT
+(
+ object_id         BIGINT(20) NOT NULL,
+ latest_move_id      BIGINT(20),
+ current_location_id  integer,
+ FOREIGN KEY(object_id) REFERENCES MUSIT_MAPPING.VIEW_MUSITTHING(id)
+);
+
+CREATE TABLE MUSARK_STORAGE.STORAGE_NODE(
+ storage_node_id BIGINT NOT NULL  AUTO_INCREMENT,
+ primary key (storage_node_id)
+);
+
+insert INTO MUSARK_STORAGE.STORAGE_NODE(storage_node_id) VALUES(3);
```

Comment:
Nitpicking here... but we should be consistent in _how_ we write SQL. Either we use UPPER_CASE for SQL keywords, or we use lower_case for keywords. Agree?